apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -347,6 +351,77 @@ static void tc_libc_signal_psignal(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn                   :tc_libc_signal_sigwait
+* @brief                :tc_libc_signal_sigwait test sigwait function
+* @scenario             :If signo < 0, it return errno by get_errno().
+*                        Else if signo > 0, return signo that signal number in the location referenced by sig.
+* API's covered         :sigwait()
+* Preconditions         :none
+* Preconditions         :none
+* @return               :void
+*/
+
+static int sigwait_receiver(int argc, FAR char *arvg[])
+{
+	sigset_t set = ALL_SIGNAL_SET;
+	int ret_chk;
+	int sig;
+
+	ret_chk = sigwait(&set, &sig);
+
+	if (ret_chk == OK && sig == SIG3) {
+		exit(0);
+	}
+	exit(1);
+}
+
+static int sigwait_sender(int argc, FAR char *arvg[])
+{
+	int ret_chk;
+
+	ret_chk = kill(atoi(arvg[argc - 1]), SIG3);
+
+	if (ret_chk == OK) {
+		exit(0);
+	}
+	exit(1);
+}
+
+static void tc_libc_signal_sigwait(void)
+{
+	int recv_pid;
+	int snd_pid;
+	FAR char const *in_buf[] = { NULL, NULL };
+	char buf[11];
+	int recv_status;
+	int snd_status;
+	int recv_ret;
+	int snd_ret;
+
+	/* sigwait_receiver wait for sigwait_sender to send a signal */
+	recv_pid = task_create("receiver", PRIORITY, STACKSIZE, sigwait_receiver, NULL);
+	if (recv_pid < 0) {
+		return;
```

Comment:
This case is also fail for testcase. Please add TC_ASSERT_XXX macro.

Reply:
Add macro